pkg/state/state_ofed.go

```diff
@@ -152,6 +239,22 @@ func (s *stateOFED) GetWatchSources() map[string]*source.Kind {
 	return wr
 }
 
+func (s *stateOFED) mountAdditionalVolumesFromConfigMap(
```

Comment:
nit: this method doesnt actually mount anything but rather generate volumes and volume mounts. also it mainly revolves around additionalVolumeMounts type so perhaps we should have a method in that struct.

suggestion:

define:
`(am *additionalMounts) FromConfigMap(cm *v1.ConfigMap, destDir) error`

which will populate additionalMounts from configmap.

then you just need to rename this method to `handleAdditionalMounts` which will get configmap and call `am.FromConfigMap(....)`


wdyt ?

Reply:
also please add docstring to added methods above :) if we go this route.